src/plugins/expressions/server/legacy.ts

```diff
@@ -112,14 +112,16 @@ export const createLegacyServerEndpoints = (
    * results back using ND-JSON.
    */
   plugins.bfetch.addBatchProcessingRoute(`/api/interpreter/fns`, request => {
-    const scopedClient = core.elasticsearch.dataClient.asScoped(request);
     const handlers = {
       environment: 'server',
       elasticsearchClient: async (
         endpoint: string,
         clientParams: Record<string, any> = {},
         options?: any
```

Comment:
Would you mind switching it to `APICaller` interface as you touch this code, please?